tests/utils.py

```diff
@@ -0,0 +1,47 @@
+import os
+import timm
+import torch
+import unittest
+
+from packaging.version import Version
```

Comment:
Seems like we have a new test dependency on packaging, this should be added to `pyproject.toml` and `requirements/tests.txt`

Reply:
Hm... I thought it's a default python package, but probably install with other dependency

Reply:
It is not default, it is installed using `pip install packaging`. It may or may not be installed, we should add an explicit dependency on it.